core/src/main/java/com/dtolabs/rundeck/core/execution/workflow/WorkflowExecutionListenerStepMetrics.java

```diff
@@ -65,4 +75,11 @@ public void beginExecuteNodeStep(ExecutionContext context, NodeStepExecutionItem
     public void finishExecuteNodeStep(NodeStepResult result, ExecutionContext context, StepExecutionItem item, INodeEntry node) {
 
     }
+
+    private void logError(String pluginName, Integer step, String message) {
+        MDC.put("pluginName",  pluginName);
```

Comment:
Why are you putting the plugin name and step in the MDC rather than outputting them directly in the log? The `message` in this instance is already being logged by a different logger in a different location, so essentially this is just duplicating current behavior.